src/config/file/v1.rs

```diff
@@ -51,6 +52,9 @@ struct RawConfigV1 {
     integrations: Vec<Integration>,
 
     debugging: Debugging,
+
+    #[allow(dead_code)]
+    version: usize,
```

Comment:
Instead, prefer:
```suggestion
    #[serde(rename(deserialize = "version"))]
    _version: usize,
```

Which denotes the variable as unused. Generally we try to avoid overriding Clippy when other options exist to resolve the issue.

Reply:
ack